app/views/attachment_mailer/new_attachment_notification.html.haml

```diff
@@ -1,23 +1,55 @@
-%table.full-width.header
-  %tr
-    %td.hero
-    %td
-      = image_tag "emails/icon-pencil-circle.png", class: "header-icon"
+- icon = "emails/icon-clippedPage.png"
+- panel_action = t("mailer.new_attachment_email.header_html", 
+                  user_name: @attachment_user.full_name,
+                  proposal_name: @proposal.name)
+- panel_action_date = @proposal.created_at.strftime("%m/%d/%Y") + " at " + @proposal.created_at.strftime("%I:%M %P")
+- panel_image = "http://fakeimg.pl/900x300?text=Preview"
+- cta_subheader = t("mailer.new_attachment_email.subheader")
 
-%table.two-thirds-width
+%table.body
   %tr
-    %td.header
-      = t("mailer.new_attachment_email.header_html",
-        user_name: @attachment_user.full_name,
-        proposal_name: @proposal.name)
+    %td.center{align: "center", valign: "top"}
+      %center
+        = render partial: "mail_shared/email_header/topbar"
+        = render partial: "mail_shared/email_header/icon"
 
-%table.two-thirds-width
-  %tr
-    %td.subheader
-      = t("mailer.new_attachment_email.subheader")
+        / Start Container
+        %table.container
+          %tr
+            %td
+              / Start Header
+              = render partial: "mail_shared/email_header/hero_text", locals: { text: t("mailer.new_attachment_email.header_html", user_name: @attachment_user.full_name)}
```

Comment:
nice -- do you want to use these partials in the other emails in this PR too, or wait until later?